pages/login.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="login.css">
+</head>
+<body>
+    <div class="container">
```

Comment:
You can make use of semantic HTML tags like `main`, `section`, `aside`, `header`, `footer`, and so on for main parts of your code for accessibility and better code readability. You can use div tags for smaller chunks of code inside the main sections.